uptane-standard.md

```diff
@@ -861,39 +801,43 @@ If a step in the following workflows does not succeed (e.g., the update is abort
-    2. Check that the Targets metadata from the Image repository matches the Targets metadata from the Director repository:
-        1. Check that the non-custom metadata (i.e., length and hashes) of the unencrypted image are the same in both sets of metadata.
-        2. Check that all "MUST match" custom metadata (e.g., hardware identifier and release counter) are the same in both sets of metadata.
-        3. Check that the release counter in the previous targets metadata file is less than or equal to the release counter in this targets metadata file.
+1. Load the repository mapping metadata ({{repo_mapping_meta}}).
+2. Refresh the top-level metadata for the Director repository, by following the steps in {{refresh_toplevel_metadata}}. The first repository listed in the repository mapping metadata is always the Director repository.
+3. Iterate through the image metadata listed in the Director repository's Targets metadata file, and **store** the image metadata this ECU is responsible for verifying. A primary ECU is responsible for verifying ALL image metadata; a secondary ECU is responsible for verifying only image metadata that includes its own ECU identifier and hardware ID.
+4. For all images this ECU is responsible for, look for matching image metadata from other repositories until the threshold listed in the repository mapping metadata is met, **or** until a terminating mapping fails. The repository mapping metadata contains a list of mappings that determine which repositories are required to have signed metadata for a particular image. For each piece of image metadata this ECU stored in the previous step, iterate through those mappings as follows:
+    1. If the name of the image currently being processed does not match the image paths listed in the current mapping, skip to the next mapping.
+    2. Following the procedure in {{refresh_toplevel_metadata}}, refresh the top-level metadata for each repository listed in this mapping that has not yet been refreshed during this run-through of full verification.
+    3. Attempt to locate metadata for the image currently being processed in each repository listed in this mapping, using the procedure for resolving delegations listed in {{resolve_delegations}}.
+    4. Determine if there is a set of matching metadata (from the metadata located in the previous step) that meets the threshold specified in the current mapping. For a set to match, the following criteria MUST be met:
+        * The non-custom metadata of the image (i.e., length and hashes) are identical.
+        * If any member of the set contains "MUST match" custom metadata (as described in {{custom_image_metadata}}), all members of the set must have identical "MUST match" custom metadata.
+        * If the custom metadata from any repository other than the Director contains an ECU identifier, it MUST be the same as the ECU identifier in the Director's custom metadata.
```

Comment:
Fair point; it's certainly only relevant in the multi-director case. My judgement was that it doesn't clutter up the procedure too much, and doesn't require explanation of the multi-director case to understand, so it could stay.